go.mod

```diff
@@ -68,6 +67,7 @@ require (
 	github.com/evanphx/json-patch/v5 v5.6.0 // indirect
 	github.com/felixge/httpsnoop v1.0.3 // indirect
 	github.com/fsnotify/fsnotify v1.6.0 // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
```

Comment:
is this a drop-in replacement? should we do a replace to get rid of indirect deps?

Reply:
Yes, see: https://github.com/kubernetes-sigs/yaml

We'd need to upgrade open-policy/agent/opa to get rid of the indirect dependency.

```
go mod why github.com/ghodss/yaml                                                         
# github.com/ghodss/yaml
github.com/open-policy-agent/gatekeeper
github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers/rego
github.com/open-policy-agent/opa/topdown
github.com/ghodss/yaml
```

Reply:
I'd rather we upstream a patch.

Reply:
This is a dependency here as well as in open-policy-agent/opa. The order in which it gets applied is not that important.